lightning-rapid-gossip-sync/src/processing.rs

```diff
@@ -194,7 +194,13 @@ impl<NG: Deref<Target=NetworkGraph<L>>, L: Deref> RapidGossipSync<NG, L> where L
 				synthetic_update.htlc_maximum_msat = htlc_maximum_msat;
 			}
 
-			network_graph.update_channel_unsigned(&synthetic_update)?;
+			match network_graph.update_channel_unsigned(&synthetic_update) {
+				Ok(_) => {},
+				Err(LightningError { action: ErrorAction::IgnoreDuplicateGossip, .. }) => {},
+				Err(LightningError { action: ErrorAction::IgnoreAndLog(_), .. }) => {},
```

Comment:
Hm, but couldn't a similar argument be made for the whole PR: that the dataset is probably bunk when we receive a duplicate update in the first place?

Reply:
I still think on client side we should ignore those errors. 
"There is something wrong with this server update" => "we should ignore and not fail client side"
Server-side should be responsible for its own correctness and should have its own checks in place. 

IIUC, ignoring the update leaves the network graph still functional instead of erroring out and stopping on some  mistake from server. (potentially out-of-date is better than errors until server fixes the potential bug)
And the way rgs currently works with latest_seen_timestamp and backdate_timestamp, it makes sense to ignore the noisy updates ? 

#1785 => As of now, RGS-client doesn't accept stale-update, it ignores the stale-update.
#1784 => As of now, RGS-client ignores the update for channel which doesn't exist and does not fail.

Reply:
"get that feedback than just silently continue" -> does debug logging with outdated network graph as feedback work? or we want to surface errors to users?

1784 -> Clarifying, i meant after this PR, we are catching and ignoring "IgnoreError" as well right? and it won't fail.